achievements/de.json

```diff
@@ -0,0 +1,30 @@
+{
+  "NEW_ACHIEVEMENT_1_0_NAME":  "Alchemie Praktikant",
+  "NEW_ACHIEVEMENT_1_0_DESC":  "Kreiere dein erstes Element mit Alchemie!",
+  "NEW_ACHIEVEMENT_1_1_NAME":  "Entdecke das Entwicklungsteam",
+  "NEW_ACHIEVEMENT_1_1_DESC":  "N/A",
+  "NEW_ACHIEVEMENT_1_2_NAME":  "Das ist eine Falle",
```

Comment:
Deepl/google translate tell me that this means "This is a trap", is this a mistake ?
In this context, plant = vegetal